### DISCUSSION TITLE
Usr point issue

### DISCUSSION THREAD

**Discussion by francky60** [flybywiresim/aircraft#2760] [Q&A] [answered]
Hello
Somebody know how to delete " usr point" created in the fpl because the plane can do whatever just after passing this waypoint
Thanks
Regards

**Comment by MapleNAV-TCASFAULT** [comment, marked as answer]
You can't unfortunately

**Comment by robinottawa** [comment]
Do what? I've never noticed a problem.

**Comment by francky60** [comment]
Hellousr point is like for exemple a discontinuity in the fpl with boeing airplane. For the moment it s not possible to delete this point with the a32nx. Therefore sometimes depending on your fpl these usr points can disturb the nav and your plane go back to the previous point.( see the vidéo from easyjetsimpilot on youtube  in attached)https://youtu.be/JoIiy6rD2-cRegardsEnvoyé depuis mon appareil Samsung









-------- Message d'origine --------
De : robinottawa <notifications@github.com> 
Date : 29/12/2020  18:26  (GMT+01:00) 
À : flybywiresim/a32nx <a32nx@noreply.github.com> 
Cc : francky60 <franck.moriere@free.fr>, Author <author@noreply.github.com> 
Objet : Re: [flybywiresim/a32nx] Usr point issue (#2760) 


Do what? I've never noticed a problem.
